tile_test.go

```diff
@@ -134,3 +115,32 @@ func TestTileZRes(t *testing.T) {
 		}
 	}
 }
+
+func TestTileToFromPixel(t *testing.T) {
+	tile := tegola.NewTile(20, 0, 0)
+	fn := func(idx int, pt [2]float64) {
+		npt, err := tile.FromPixel(tegola.WebMercator, pt)
+		if err != nil {
+			t.Errorf("[%v] Error, Expected nil Got %v", idx, err)
+			return
+		}
+		gpt, err := tile.ToPixel(tegola.WebMercator, npt)
+		if err != nil {
+			t.Errorf("[%v] Error, Expected nil Got %v", idx, err)
+			return
+		}
+		// TODO: gdey need to find the utility math function for comparing floats.
+		if pt[0] != gpt[0] {
```

Comment:
you can always use reflect.DeepEqual(), though it's a bit overkill for this use case.

Reply:
This is to compare floats taking into account significant digits. DeepEqual would work just as well as comparing them directly. Right now this test could fail due to rounding errors; due to significant digits.